Source/Core/DolphinQt2/GameList/TableProxyModel.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/GameList/TableProxyModel.h"
+#include "DolphinQt2/GameList/GameListModel.h"
+
+TableProxyModel::TableProxyModel(QObject* parent) : QSortFilterProxyModel(parent)
+{
+  setSortCaseSensitivity(Qt::CaseInsensitive);
+  sort(GameListModel::COL_TITLE);
```

Comment:
You're correct, I'm amending it like you say, I'm trying to handle the lint errors but the lint.sh output seems to be identical to what I have committed, so I am not sure whats wrong with the formatting on the maps.